lib/cli/cli.js

```diff
@@ -12,9 +12,12 @@ var emberCLIVersion      = versionUtils.emberCLIVersion;
 var InstallationChecker  = require('../models/installation-checker');
 
 function CLI(options) {
+  this.name = options.name;
   this.ui = options.ui;
   this.analytics = options.analytics;
   this.testing = options.testing;
+  this.cliRoot = options.cliRoot;
```

Comment:
I won't have time to do the changes today, so if you want to do them and merge the PR, go ahead.